runtimes/nodejs/nodejs.md

```diff
@@ -21,38 +21,33 @@ Node.js is a platform built on Chrome's JavaScript runtime for building fast, sc
 
 Refer to the page [Deploy an application on Clever Cloud](/doc/clever-cloud-overview/add-application/).
 
-## Necessary information
+## Requirements
 
 Be sure that:
 
 * you have pushed in <b>master branch</b>
 * you listen on <b>port 8080</b>
-* you have added application <b>name</b> in package.json
-* you have added application <b>version</b> in package.json
 * you have added application <b>start script</b> in package.json
```

Comment:
Yes it does. Maybe I should extend to "either add a scripts.start or name field"